definitions/tools/cnvkit_batch.cwl

```diff
@@ -13,16 +13,19 @@ requirements:
       ramMin: 4000
       tmpdirMin: 10000
     - class: InlineJavascriptRequirement
+arguments:
+    ["--normal"]
```

Comment:
Does this work to have this repeated here and for the `normal_bam` input?  The help text makes it sound like it wants to be followed by 0 or more files, but it looks a little strange to specify it twice.

Reply:
I think it's okay to have `--normal` `other inputs` `--normal actual_normal.bam`. I will double check. 
I'm sure there is a way to force the `--normal` flag to be passed even if no normal bam input is provided. Adding it to the arguments was the easiest for me.

Reply:
One way is to keep it in arguments like you have and then make sure the `normal_bam` input is first after the arguments (with no specified prefix).  I think right now you'll end up with `--normal --normal x.bam` as it is written.

Reply:
ahh what a good solution!